docs/source/getting-started/installing.adoc

```diff
@@ -45,6 +45,12 @@ https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/e
 https://msdn.microsoft.com/en-us/virtualization/hyperv_on_windows/quick_start/walkthrough_virtual_switch[add a Virtual Switch]
 using the Hyper-V Manager. Make sure that you pair the virtual switch
 with a __network card (wired or wireless) that is connected to the network__.
+Minishift tries per default to use the first virtual switch it detects.
+This can lead to conflicts where this switch is used by another application.
```

Comment:
Does this mean one switch is used for one VM? AFAIK Minishift would work fine as long as the virtual switch is an external type switch.